src/modules/fancyzones/editor/FancyZonesEditor/Models/Settings.cs

```diff
@@ -43,7 +47,11 @@ private enum CmdArgs
         public const ushort _blankCustomModelId = 0xFFFA;
         public const ushort _lastDefinedId = _blankCustomModelId;
 
-        private const int _defaultDPI = 96;
+        private const string ZonesSettingsFile = "\\Microsoft\\PowerToys\\FancyZones\\zones-settings.json";
+
+        private const string ActiveZoneSetsTmpFileName = "FancyZonesActiveZoneSets.json";
+        private const string AppliedZoneSetsTmpFileName = "FancyZonesAppliedZoneSets.json";
+        private const string DeletedCustomZoneSetsTmpFileName = "FancyZonesDeletedCustomZoneSets.json";

```

Comment:
Add a comment to clarify these are not localizable strings.

Reply:
Done